Directory.Build.props

```diff
@@ -5,14 +5,14 @@
     <Authors>$(Company)</Authors>
     <Copyright>Copyright © $(Company) $([System.DateTime]::Now.Year)</Copyright>
     <Trademark>$(Company)™</Trademark>
-    <VersionPrefix>1.0.0</VersionPrefix>
+    <VersionPrefix>2.0.1</VersionPrefix>
```

Comment:
This will be a brand new package [that doesn't exist yet](https://www.nuget.org/packages?q=Kentico.Xperience.Siteimprove) so we can leave the version at `1.0.0` until we publish.

We also might publish a "pre-release" package first.